lte/protos/session_manager.proto

```diff
@@ -1014,37 +1048,54 @@ message UPFAssociationState {
 
 message UPFFeatureSet {
   // 15.8 (Sec 8.2.25) :  UP Function Features
-  bool downlink_data_buffering_cp = 1;   // Downlink Data Buffering in CP function supported in UPF
+  // Downlink Data Buffering in CP function supported in UPF
+  bool downlink_data_buffering_cp = 1;   
+  // Buffering parameter, Downlink DataNotification Delay supported by UPF
   bool downlink_data_notification_delay = 2;
-                                           // Buffering parameter, Downlink DataNotification Delay supported by UPF
-  bool dl_buffering_duration = 3;        // Buffering parameter, DL Buffering Duration is supported by UPF
-  bool traffic_steering = 4;             // Traffic Steering is supported by the UPF
-  bool f_teid = 5;                       // F-TEID allocation/release in the UP function is supported by UPF
-  bool pfd_management = 6;               // PFD Management is supported by UPF
-  bool header_enrichment_uplink = 7;     // Header enrichment of Uplink Traffic is supported by UPF
+  // Buffering parameter, DL Buffering Duration is supported by UPF  
+  bool dl_buffering_duration = 3;        
+  // Traffic Steering is supported by the UPF
+  bool traffic_steering = 4;             
+  // F-TEID allocation/release in the UP function is supported by UPF
+  bool f_teid = 5;                       
+  // PFD Management is supported by UPF
+  bool pfd_management = 6;               
+  // Header enrichment of Uplink Traffic is supported by UPF
+  bool header_enrichment_uplink = 7;     
+  // Traffic Redirection Enforcement in the UP function is supported in UPF
   bool traffic_redirection_enforcement = 8;
-                                         // Traffic Redirection Enforcement in the UP function is supported in UPF
-  bool end_of_marker_packet = 9;         // Sending End of Marker packets supported by the UPF
-  bool pdi_optimization_signalling = 10; // Support PDI optimized signalling in UPF
-  bool ul_dl_buffering_control = 11;     // Support of UL/DL Buffering Control
-  bool quota_application_to_apply = 12;  // The UPF supports being provisioned with the Quota Action to apply
-  bool trace = 13;                       // The UPF supports Trace
-  bool frame_routing = 14;               // UPF supports Framed Routing
+  // Sending End of Marker packets supported by the UPF
+  bool end_of_marker_packet = 9;         
+  // Support PDI optimized signalling in UPF
```

Comment:
"signalling" is a misspelling of "signaling"

Reply:
resolved